launcher/ui/pages/instance/VersionPage.cpp

```diff
@@ -707,6 +723,19 @@ void VersionPage::on_actionRevert_triggered()
     {
         return;
     }
+    auto component = m_profile->getComponent(version);
+
+    auto response = CustomMessageBox::selectable(this, tr("CAREFUL!"),
```

Comment:
```suggestion
    auto response = CustomMessageBox::selectable(this, tr("Confirm reversion"),
```

i almost said deletion again lol